plugins/serializers/extr/extr.go

```diff
@@ -315,3 +303,81 @@ func firstCharToLower(str string) string {
 
 	return ""
 }
+
+func buildMap(startMap map[string]interface{}, startKey string, s []string, numKeys int) (
+	map[string]interface{}, string, error) {
+
+	sIndex := numKeys - 1
```

Comment:
This may be moved inside the "else" clause because if "numKeys == 0" it will return not needing the sIndex.